src/website/app/demos/Button/bestPractices.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Button from '../../../../Button';
+
+export default [
+  {
+    type: 'do',
+    title: 'use the appropriate variant for your intent',
+    example: (
+      <Button variant="warning" primary>
+        Launch Kittens into Space
```

Comment:
We should attempt to use a common tone and common similar examples styles across the website.   I'd suggest to focus on conveying a message rather than humor.